packages/cli/src/databases/migrations/common/1728396464278-AddMissingPrimaryKeyOnAnnotationTagMapping.ts

```diff
@@ -0,0 +1,25 @@
+import { ApplicationError } from 'n8n-workflow';
+
+import type { IrreversibleMigration, MigrationContext } from '@/databases/types';
+
+export class AddMissingPrimaryKeyOnAnnotationTagMapping1728396464278
+	implements IrreversibleMigration
+{
+	async up({ queryRunner, tablePrefix }: MigrationContext) {
+		// Check if the primary key already exists
+		const table = await queryRunner.getTable(`${tablePrefix}execution_annotation_tags`);
+
+		if (!table) {
+			throw new ApplicationError('execution_annotation_tags table not found');
+		}
```

Comment:
This should never happen right? I think we could `assert(table, 'execution_annotation_tags table not found')` imported from `node:assert`